upup/pkg/fi/cloudup/hetzner/cloud.go

```diff
@@ -41,6 +41,7 @@ const (
 	TagKubernetesInstanceUserData    = "kops.k8s.io/instance-userdata"
 	TagKubernetesInstanceNeedsUpdate = "kops.k8s.io/needs-update"
 	TagKubernetesVolumeRole          = "kops.k8s.io/volume-role"
+	TagKubernetesNodeLabelPrefix     = "kops.k8s.io/node-label/"
```

Comment:
Hetzner does not allow multiple slashes for labels:
> Valid label keys have two segments: an optional prefix and name, separated by a slash ( / ).

https://docs.hetzner.cloud/#labels

```suggestion
	TagKubernetesNodeLabelPrefix     = "kops.k8s.io/node-label."
```

Reply:
this wont work because the node labels themselves may have slashes:

`kops.k8s.io/node-label.node-role.kubernetes.io/node`

I just pushed a potential fix. Its not great but given the label requirements I'm not sure we can do any better.